conf/bk_server.conf

```diff
@@ -358,6 +358,11 @@ journalDirectories=/tmp/bk-txn
 # Set the size of the journal queue.
 # journalQueueSize=10000
 
+# Set the max amount of memory that can be used by the journal.
+# If empty, this will be set to use 5% of available direct memory
+# Setting it to 0, it will disable the max memory control for the journal.
+# journalMaxMemorySizeMb=
```

Comment:
So looks like this will be additional limit to t he max Journal Q size?

Reply:
Correct. that would still be in place